File: app/src/main/java/com/rodionov/igor/mobiletest/model/Category.kt
package com.rodionov.igor.mobiletest.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "categories")
class Category {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Int
    @ColumnInfo(name = "custom_id")
    var _id: String;
    @ColumnInfo(name = "updated_at")
    var updated_at: String
    @ColumnInfo(name = "slug")
    var slug: String
    @ColumnInfo(name = "custom_module_eid")
    var custom_module_eid: String
    @ColumnInfo(name = "eid")
    var eid: String
    @ColumnInfo(name = "title")
    var title: String
    @ColumnInfo(name = "description")
    var description: String
    @ColumnInfo(name = "version")
    var __v: Int = 0
    @ColumnInfo(name = "active")
    var _active: Boolean = false
    @ColumnInfo(name = "created_at")
    var created_at: String

    init {
        id = 0
        _id = ""
        updated_at = ""
        slug = ""
        custom_module_eid = ""
        eid = ""
        title = ""
        description = ""
        created_at = ""
    }
}

File: QuickSeries/QuickSeries/Models/Category.swift



import Foundation
struct Category : Codable {
	let _id : String?
	let updated_at : String?
	let slug : String?
	let custom_module_eid : String?
	let eid : String?
	let title : String?
	let description : String?
	let __v : Int?
	let _active : Bool?
	let created_at : String?

	enum CodingKeys: String, CodingKey {

		case _id = "_id"
		case updated_at = "updated_at"
		case slug = "slug"
		case custom_module_eid = "custom_module_eid"
		case eid = "eid"
		case title = "title"
		case description = "description"
		case __v = "__v"
		case _active = "_active"
		case created_at = "created_at"
	}

	init(from decoder: Decoder) throws {
		let values = try decoder.container(keyedBy: CodingKeys.self)
		_id = try values.decodeIfPresent(String.self, forKey: ._id)
		updated_at = try values.decodeIfPresent(String.self, forKey: .updated_at)
		slug = try values.decodeIfPresent(String.self, forKey: .slug)
		custom_module_eid = try values.decodeIfPresent(String.self, forKey: .custom_module_eid)
		eid = try values.decodeIfPresent(String.self, forKey: .eid)
		title = try values.decodeIfPresent(String.self, forKey: .title)
		description = try values.decodeIfPresent(String.self, forKey: .description)
		__v = try values.decodeIfPresent(Int.self, forKey: .__v)
		_active = try values.decodeIfPresent(Bool.self, forKey: ._active)
		created_at = try values.decodeIfPresent(String.self, forKey: .created_at)
	}

}
